internal/engine/execution/queries.go

```diff
@@ -96,7 +98,7 @@ var (
 	// sqlBackfillSchemaTableV1 adds the owner and name to all existing schemas,
 	// and updates the version to 1.
 	sqlBackfillSchemaTableV1 = fmt.Sprintf(`
-	UPDATE %s.kwil_schemas SET owner = $1, name = $2, version = 1;
+	UPDATE %s.kwil_schemas SET owner = $1, name = $2, version = 1 WHERE dbid = $3;
```

Comment:
Oh yeah, I thought this was in the last PR.

Reply:
Could've sworn it was. A lot of this has sort've been continual work, as the first few days of foreign calls were being worked on while procedures was in review.

Probably just got my wires crossed here